source/includes/extracts-query-bar.yaml

```diff
@@ -4,14 +4,40 @@ content: |
    
    For details on sampling, see the :ref:`FAQ <compass-faq-sampling>`. 
 ---
-ref: query-bar-type
+ref: query-bar-type-schema
 content: |
    You can type MongoDB queries into the query bar. In the query
    expression, you can use the various MongoDB :manual:`query operators
    </reference/operator/query/>` except for the :query:`$text` query
    operator.
 
-   As you type, the :guilabel:`Apply` button is disabled and the
+   As you type, the :guilabel:`Analyze` button is disabled and the
+   :guilabel:`Filter` label turns red until a valid query is entered.
```

Comment:
I would break the sentence at "disabled" and start with "The Filter label appears red until..." ("turns" implies that it started as black then became red.)